code/game/objects/items/GriffeningPorted.dm

```diff
@@ -0,0 +1,805 @@
+GLOBAL_LIST(CardCreatures)
+GLOBAL_LIST(CardEffects)
+GLOBAL_LIST(CardEquipment)
+GLOBAL_LIST(CardAreas)
+
+/obj/item/griffeningdeck
+	name = "deck of griffening cards"
+	desc = "A deck of griffening playing cards."
+	icon = 'icons/obj/toy.dmi'
+	var/deckstyle = "nanotrasen"
+	icon_state = "deck_nanotrasen_full"
+	w_class = WEIGHT_CLASS_SMALL
+	var/amountofcards //I hate this
+	var/IsBooster = FALSE //Muh booster packs
```

Comment:
wtf make boosters a child type. Don't put the logic on the base type, that's stupid as fuck